grails-app/views/species/show.gsp

```diff
@@ -46,7 +46,7 @@
     </head>
 
     <body>
-        <div class="container-fluid page-taxon">
```

Comment:
We can't wrap everything in container, because it adds a whole lot of extra padding. One way would be to go with "expected" structure

```html
<div class="container">
   <div class="row">
      <div class="col-md-12">
        <header />
        <content />
      </div>
```
But that's just stupid boilerplate. The only thing a fluid container does is cancel margins from rows or something. IMHO, once we've started laying stuff out without bootstrap, the contract with bs is already out. Erurikkus isn't made as a series of rows and columns, so why not throw out the container as well.

That ended up kinda rantish, sorry. I guess the main point is that it works without container.